plugins/modules/one_image.py

```diff
@@ -462,7 +462,7 @@ def enable_image(self, image, enable):
         if changed and not self.module.check_mode:
             self.one.image.enable(image.ID, enable)
 
-        result = OpenNebulaModule.get_image_info(image)
+        result = self.OpenNebulaModule.get_image_info(image)
```

Comment:
Sorry, I was in a rush the other day, the actual thing should be:
```suggestion
        result = self.get_image_info(image)
```

Reply:
Ok, so I guess no new syntax ;)